test/integration/etcd_storage_path_test.go

```diff
@@ -851,10 +851,12 @@ func testEtcdStoragePath(t *testing.T, etcdServer *etcdtest.EtcdTestServer, gett
 	}
 	masterConfig.EtcdClientInfo.URLs[0] = testutil.GetEtcdURL()
 
-	kubeConfigFile, err := testserver.StartConfiguredMasterAPI(masterConfig)
+	_, err = testserver.StartConfiguredMasterAPI(masterConfig)
 	if err != nil {
 		t.Fatalf("error starting server: %#v", err)
 	}
+	kubeConfigFile := masterConfig.MasterClients.OpenShiftLoopbackKubeConfig
```

Comment:
This needs a comment about how `system:masters` is special and in sync between kube and origin RBAC.